api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -66,9 +66,19 @@ type VolumeReplicationGroupSpec struct {
 	// that needs to be replicated to the peer cluster.
 	PVCSelector metav1.LabelSelector `json:"pvcSelector"`
 
-	// ReplicationClass of all volumes in this replication group;
-	// this value is propagated to children VolumeReplication CRs
-	VolumeReplicationClass string `json:"volumeReplicationClass"`
+	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
(nit) We can drop this comment, there is one at the head of the file if I remember right, and comments here get into the CRD documentation, so let's keep this area clean of code comments.

Reply:
ACK